### ISSUE TITLE
monoallelic_autosomal        

### ISSUE THREAD

**Issue by angharadroberts** [obophenotype/human-phenotype-ontology#7493]
**Preferred term label:**
monoallelic_autosomal        

**Synonyms**
Autosomal Dominant 
HP:0000006

**Definition (free text, please give PubMed ID)**
The allelic requirement for autosomal dominant inheritance

**Parent term (use hpo.jax.org/app)**
allelic requirement (requested)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@angharadroberts Just to clarify, are you suggesting that we make monoallelic_autosomal a synonym of [Autosomal dominant inheritance HP:0000006](https://hpo.jax.org/app/browse/term/HP:0000006)? 
Or are you suggesting a new separate hierarchy called "allelic requirement"? What is the additional utility of this compared to the existing terms?

**Comment by pnrobinson** [comment]
sorry, just seeing this https://github.com/obophenotype/human-phenotype-ontology/issues/7492

**Comment by pnrobinson** [comment]
@angharadroberts added this as synoynm type "allelic requirement"
